### DISCUSSION TITLE
procedural terrain?

### DISCUSSION THREAD

**Discussion by jasonswearingen** [spimort/TerraBrush#81] [Q&A]
Hello, is it possible to use TerraBrush to procedurally create/modify terrain at runtime?   If so, could I get some pointers on where to start looking into this?  (I am a good csharp dev, but new to godot)

edit:  The discussion on loading heightmaps/GIS seems to be a good starting point for me, so I'll try to recreate that for now. https://github.com/spimort/TerraBrush/discussions/72

**Comment by spimort** [comment]
Awesome! I'll close this one then in favor of #72 ! 🙂
Feel free to continue the discussion in the other thread!
